<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageDexDeposit</class>
 <widget class="QWidget" name="PageDexDeposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;
color:#666666;</string>
  </property>
  <widget class="QLabel" name="depositAddressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Deposit address</string>
   </property>
  </widget>
  <widget class="QLabel" name="providerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Provider: native</string>
   </property>
  </widget>
  <widget class="QLabel" name="depositFeeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Deposit Fee: 0.00001 ETH</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Confirmation: 3 confirmations</string>
   </property>
  </widget>
  <widget class="QLabel" name="contractLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Contract:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>287</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-width: 1px;
	border-radius: 5px;
	border-color: lightgrey;
    padding-right: 3px;
    padding-left: 3px;
	color:black;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="depositAddressPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-top:0px;
padding-bottom:0px;
padding-left:0px;
padding-right:0px;
color:black;
text-align:right;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="tickerLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:black;</string>
   </property>
   <property name="text">
    <string>ETH</string>
   </property>
  </widget>
  <widget class="QLabel" name="minimalDepositLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Minimal Deposit: 0.01 ETH</string>
   </property>
  </widget>
  <widget class="QLabel" name="depositFundsForLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Deposit funds for: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../res.qrc">:/res/ic/res/ic/arrowLeftDown.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="contractAddressPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-top:0px;
padding-bottom:0px;
padding-left:0px;
padding-right:0px;
color:black;
text-align:right;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
